mttl/models/containers/selectors/arrow_selector.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass
+
+from mttl.models.containers.selectors.base_selectors import (
+    PerTokenSelector,
+    PerTokenSelectorConfig,
+    register_multi_expert_selector,
+)
+
+
+def comput_arrow_embeddings(
+    library,
+    selector_data_id=None,
+    ab_only=True,
+    tie_params=None,
+    tie_op="concat",
+    base_model_proto=False,
+    recompute_prototypes=False,
+) -> str:
+    from mttl.models.library.library_transforms import ArrowConfig, ArrowTransform
+
+    cfg = ArrowConfig(
+        name=selector_data_id,
+        ab_only=ab_only,
+        tie_params=tie_params or "default",
+        tie_op=tie_op,
+    )
+    ArrowTransform(cfg).transform(
```

Comment:
we should probably hardcode `persist=True`

Reply:
ok good idea